README.md

```diff
@@ -283,7 +283,7 @@ If you need to setup a redirect from Docs 1.0 to Docs 2.0 manually, this is the
 
 # MDX Format
 
-Documentation must be formatted as an [MDX file](https://www.gatsbyjs.com/docs/mdx/writing-pages/) with the `.mdx` extension. MDX is a superset of [Markdown](https://www.markdownguide.org/).
+Documentation must be formatted as an [MDX file](https://mdxjs.com/docs/what-is-mdx/) with the `.mdx` extension. MDX is a superset of [Markdown](https://www.markdownguide.org/).
```

Comment:
```suggestion
Documentation must be formatted as an [MDX file](https://www.gatsbyjs.com/docs/how-to/routing/mdx/#writing-pages-in-mdx) with the `.mdx` extension. MDX is a superset of [Markdown](https://www.markdownguide.org/).
```

We're MDX in Gatsby so not quite MDX pure. Hence a change of link.

Reply:
Good to know. I found the other link more helpful to get started with MDX, I'll rework that part to include both links.